deployments/scripts/env.sh

```
@@ -4,5 +4,5 @@ CLUSTER_NAME="openchoreo-local-setup"
 CLUSTER_CONTEXT="k3d-${CLUSTER_NAME}"
 
 # WSO2 API Platform / Gateway Operator versions
-GATEWAY_OPERATOR_VERSION="0.6.0"
-GATEWAY_CHART_VERSION="1.0.1"
+GATEWAY_OPERATOR_VERSION="0.7.0"
```

Shellcheck (0.11.0)

[warning] GATEWAY_OPERATOR_VERSION appears unused. Verify use (or export if used externally).

(SC2034)